cms/static/js/views/move_xblock_breadcrumb.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * MoveXBlockBreadcrumb shows back button and breadcrumb to move back to parent.
```

Comment:
Description needs to changed after back button is removed and there is no back option, it should reflect something like move to parent locations etc.